src/js/components/Home/Countries.js

```diff
@@ -2,22 +2,44 @@ import React, {Component} from "react";
 import Map from "./Map";
 import './MapStyles.sass'
 import ReactFlagsSelect from 'react-flags-select';
+import Geocode from "react-geocode";
 
+// function onSelectFlag(countryCode){             //TODO: Function to be implemented
+//     console.log(countryCode)
+// }
+// set Google Maps Geocoding API for purposes of quota management. Its optional but recommended.
+Geocode.setApiKey("AIzaSyBiCz9njYP8nYxEUEQLzltCM8kAnm7z6To");
 
 class Mapbuttons extends Component {
```

Comment:
Inspect your page and see if you get warnings when calling Component.
If you do, extend React.Component.